releasenotes/notes/0.19/added-multiformat-support-b5d3c7c7c1536951.yaml

```diff
@@ -0,0 +1,22 @@
+---
+features:
+  - |
+    Using :meth:`.QuantumCircuit.draw` with the ``latex`` drawer will now generate a file
+    in an image format inferred from the filename extension, for example::
+
+      import qiskit
+
+      circuit = qiskit.QuantumCircuit(2)
+      circuit.h(0)
+      circuit.cx(0, 1)
+      circuit.draw('latex', filename='./file.jpg')
+    
+    This will save the circuit in the JPEG format (earlier it always used to save it in PNG).
+    Refer to `#6448 <https://github.com/Qiskit/qiskit-terra/issues/6448>`__ for more details.
+
+    Now, if it encounters a filename extension which is not supported, for example::
+      
+      circuit.draw('latex', filename='./file.spooky')
+
+    it will give user a ``ValueError`` to change the filename extension to a supported image format.
+    A similar ``ValueError`` is raised if a user tries to save the output of the ``text`` drawer to a file without using a ``.txt`` extension.
```

Comment:
I feel like this should also be an upgrade note as it's a potentially breaking change. Especially the text drawer one, I'm tempted to revert that because it seems overly pedantic to me to force people to use a `.txt` extension for a text file.

Reply:
Yeah, that's exactly what I thought about the `txt` extension when I read this note as well...

Reply:
Pushed 2c93a9a46 to revert the `txt` change.